meta-resin-common/classes/resin-connectable-service.bbclass

```diff
@@ -0,0 +1,38 @@
+# resin-connectable-service bbclass
+#
+# Author: Andrei Gherzan <andrei@resin.io>
+
+python __anonymous() {
+    # Enable/disable systemd services listed in RESIN_CONNECTABLE_SERVICES based on
+    # RESIN_CONNECTABLE_ENABLE_SERVICES
+
+    pn = d.getVar('PN', True)
+    services = d.getVar('RESIN_CONNECTABLE_SERVICES', True).split()
+    resin_connectable = d.getVar('RESIN_CONNECTABLE', True)
+    resin_connectable_enable_services = d.getVar('RESIN_CONNECTABLE_ENABLE_SERVICES', True)
+
+    if resin_connectable == '1' and pn in services:
+        if resin_connectable_enable_services == '1':
+            d.setVar('SYSTEMD_AUTO_ENABLE', 'enable')
```

Comment:
Indeed but we force enable even if other configuration was made (ex bbappend or conf). As well is better to not reside on a default that might change.